change/@fluentui-react-dialog-396e0302-47d0-4e6f-84ab-c29537896234.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 44 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27081&commitId=423130db7b24f969b301efe584d91a14b4125b46&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1824 screenshots unchanged, 0 screenshots with different dimensions and 44 screenshots with visible difference.


  <details><summary><b>unknown</b> 44 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.actions position start & position end - Dark Mode.chromium.png|810616| Changed|
|Dialog.actions position start & position end - High Contrast.chromium.png|833866| Changed|
|Dialog.actions position start & position end - RTL.chromium.png|913411| Changed|
|Dialog.actions position start & position end.chromium.png|913412| Changed|
|Dialog.actions position start - Dark Mode.chromium.png|810257| Changed|
|Dialog.actions position start - High Contrast.chromium.png|833479| Changed|
|Dialog.actions position start - RTL.chromium.png|913743| Changed|
|Dialog.actions position start.chromium.png|913744| Changed|
|Dialog.alert - Dark Mode.chromium.png|810257| Changed|
|Dialog.alert - High Contrast.chromium.png|833477| Changed|
|Dialog.alert - RTL.chromium.png|913743| Changed|
|Dialog.alert.chromium.png|913744| Changed|
|Dialog.default - Dark Mode.chromium.png|810257| Changed|
|Dialog.default - High Contrast.chromium.png|833477| Changed|
|Dialog.default - RTL.chromium.png|913743| Changed|
|Dialog.default.chromium.png|913744| Changed|
|Dialog.nested - Dark Mode.chromium.png|813312| Changed|
|Dialog.nested - High Contrast.chromium.png|835010| Changed|
|Dialog.nested - RTL.chromium.png|908834| Changed|
|Dialog.nested.chromium.png|908834| Changed|
|Dialog.no actions - Dark Mode.chromium.png|818158| Changed|
|Dialog.no actions - High Contrast.chromium.png|837204| Changed|
|Dialog.no actions - RTL.chromium.png|908568| Changed|
|Dialog.no actions.chromium.png|908568| Changed|
|Dialog.no title & no actions - Dark Mode.chromium.png|825529| Changed|
|Dialog.no title & no actions - High Contrast.chromium.png|842452| Changed|
|Dialog.no title & no actions - RTL.chromium.png|902475| Changed|
|Dialog.no title & no actions.chromium.png|902476| Changed|
|Dialog.no title - Dark Mode.chromium.png|817628| Changed|
|Dialog.no title - High Contrast.chromium.png|838726| Changed|
|Dialog.no title - RTL.chromium.png|907650| Changed|
|Dialog.no title.chromium.png|907651| Changed|
|Dialog.non-modal - Dark Mode.chromium.png|1466253| Changed|
|Dialog.non-modal - High Contrast.chromium.png|1491070| Changed|
|Dialog.non-modal - RTL.chromium.png|1597111| Changed|
|Dialog.non-modal.chromium.png|1597113| Changed|
|Dialog.scroll long content - Dark Mode.chromium.png|665839| Changed|
|Dialog.scroll long content - High Contrast.chromium.png|704720| Changed|
|Dialog.scroll long content - RTL.chromium.png|1052799| Changed|
|Dialog.scroll long content.chromium.png|1052823| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|817381| Changed|
|Dialog.title custom action - High Contrast.chromium.png|836611| Changed|
|Dialog.title custom action - RTL.chromium.png|909167| Changed|
|Dialog.title custom action.chromium.png|909167| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>